### DISCUSSION TITLE
[Idea] Add an optional limit to the number of orders displayed in the customer admin page

### DISCUSSION THREAD

**Discussion by Touxten** [PrestaShop/PrestaShop#35878] [Ideas]
### The problem

When customers exceed a certain number of orders, the customer page can become very slow.

### The solution

Add an optional limit to the number of orders displayed in the customer admin page

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Hlavtox** [comment]
That's already resolved on develop branch, I added pagination and limited it to 10 orders. 👍

**Reply by Touxten** [reply]
Super thanks a lot for your great work ! 

You are the best

I close the discussion
